README.md

```diff
@@ -1,19 +1,26 @@
-# numba-rvsdg
+# numba-scfg
 
-Numba compatible RVSDG (Regionalized Value State Dependence Graph)  utilities.
+Numba compatible SCFG (Structured Control Flow Graph)  utilities.
 
 ## About
 
-This repository contains Numba compatible utilities for working with RVSDGs
-(Regionalized Value State Dependency Graphs). RVSDGs are a type of
-Intermediary Representation (IR) suitable for regularizing Python bytecode
-within Numba.
+This repository contains Numba compatible utilities for working with SCFG
+(Structured Control FLow Graphs). SCFGs are a type of Intermediary
+Representation (IR) suitable for regularizing Python source- and bytecode.
 
 The code in this repository is an implementation of the CFG restructuring
 algorithms in Bahmann 2015, specifically those from section 4.1 and 4.2: namely
 "loop restructuring" and "branch restructuring". These are interesting for
-Numba because they serve to clearly identify regions within the Python
-bytecode.
+Numba because they serve to clearly identify regions within the Python source-
+and bytecode.
+
+Note: The project was previously known and `numba-rvsdg` and was renamed to
+`numba-scfg` in early 2025. The original scope was to implement Regional Value
+State Dependence Graphs (RVSDG), where SCFGs are necessary intermediary for the
+transformation from Python to RVSDG. But over time it became evident that the
+full RVSDG representation wasn't needed and so this package implements the SCFG
```

Comment:
> wasn't needed

It's still needed in a later point. I'll say SCFG is interesting enough to deserve it's own package. Not everyone needs RVSDG, but the SCFG is a useful thing for many.

Reply:
I've reworded the historical note in: dcb8f619ca7d669493a71ca59569d29374b0fd6b